lib/mobile_robot_simulator/robot_models/ackerman_robot.py

```diff
@@ -16,21 +16,24 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-#ackerman robot Class.
+# Ackerman robot Class.
```

Comment:
This comment should be the Ackerman robot class's docstring: https://www.python.org/dev/peps/pep-0257/